src/app/containers/Timestamp/timestampUtilities.js

```diff
@@ -14,3 +15,13 @@ export const formatUnixTimestamp = (timestamp, momentString, timezone) =>
   moment(timestamp)
     .tz(timezone)
     .format(momentString);
+
+export const showRelativeTime = (timestamp, isRelative, format, timezone) => {
+  if (isRelative) {
+    return relativeTime(timestamp);
+  }
+  if (!isRelative && format) {
```

Comment:
As `isRelative` is checked above and returned out of, it doesnt need to be checked here :)